src/index.html

```diff
@@ -4,7 +4,8 @@
 <head>
   <meta charset="UTF-8">
   <title>Title</title>
-  <script src="https://cdnapisec.kaltura.com/p/2215841/embedPlaykitJs/uiconf_id/42237501/"></script>
+  <script src="https://raw.githack.com/kaltura/kaltura-player-js/master/dist/kaltura-ovp-player.js"></script>
```

Comment:
index.html is a demo page ? why not use a uiConf of a player ? the master player won't always be stable

Reply:
If our master is unstable, we are in serious trouble.
Addressing a specific uוconf is much more unstable (it can changed or deleted) when fetching from the master it allows us to isolate only the specific module and load a clean player

Reply:
fixed